src/helpers/utils.js

```diff
@@ -1,14 +1,9 @@
 import { router } from '../router'
```

Comment:
Why you not create test for theses cases ?

Reply:
Im waiting this branch to be merged for create the tests on this other branch [feat-create-tests](https://github.com/amanda-yosh/the-shire/pull/9)